en_US/messages.json

```diff
@@ -134,5 +134,9 @@
           "example" : "100"
         }
       }
+    },
+    "sslIncompatibility": {
```

Comment:
I suggest to call it tlsIncompatibility instead. I know, Firefox itself still has a lot of references to "SSL" in internal option names (probably for historical reasons) but in fact Firefox does not support any version of "SSL" anymore. It's called TLS since 1999 and while it's probably not important enough to change old code I think that at least new code should not increase the usage of the legacy name. See also https://en.wikipedia.org/wiki/Transport_Layer_Security.

Reply:
thanks for the suggestion!